packages/cozy-client/types/types.d.ts

```diff
@@ -805,6 +806,45 @@ export type FolderDocument = {
  * - An io.cozy.files document
  */
 export type IOCozyFolder = CozyClientDocument & FolderDocument;
+/**
+ * - An io.cozy.remote.nextcloud document after normalization
+ */
+export type NextcloudFile = {
+    /**
+     * - Id of the file
+     */
+    _id?: string;
```

Comment:
_id is mandatory,no ?

Reply:
good point, it's mandatory (https://github.com/cozy/cozy-client/pull/1490)